rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -146,11 +150,21 @@ To view the test results and to download an analysis package:
 
 == Deploy a Process Automation
 
-Transition the project to the production phase to deploy a scheduled run configuration to a bot. The process runs fully automated from now on.
+Transition the project to the production phase to deploy a run configuration to a bot. The process runs fully automated from now on.
+
+Sometimes it's necessary to start a process manually when something happens that can only a human can evaluate. For example, the goods inwards inspection process can only start after the banana delivery ship reaches the pier wall. For this purpose, use the *Run Now* option.
+
+You have three options how to run processes:
+
+* Create and deploy a **Scheduled Run Configuration** to run the process automatically at defined times. 
+* Create and publish an **Invocable Run Configuration** to use it within other Salesforce products.
+* Create an **On-Demand Run Configuration** to start the process run manually.
```

Comment:
This text is a candidate for reuse.

Reply:
Do you think I can use it for all three paragraphs?